### ISSUE TITLE
Trained models

### ISSUE THREAD

**Issue by GreenTeaHua** [toandaominh1997/EfficientDet.Pytorch#1]
1. is there any Trained models  for test？
2. Can you reproduce the results reported in EfficientDet paper?
thanks very much.

**Comment by toandaominh1997** [comment]
thanks for your questioin @GreenTeaHua 
I will public weight, data, performance soon.

**Comment by alen-mask** [comment]
> thanks for your questioin @GreenTeaHua
> I will public weight, data, performance soon.

Hello,

any update here? will the coco-pretrained model be released?
